### PR TITLE
RF-13964 Pom Updates

### PR DESCRIPTION
**Pull request by hazendaz**

### CODE REVIEW

**Comment by hazendaz** [comment]
I've been keeping this rebased and was wondering what the chances are of this pull request being accepted.

**Comment by michpetrov** [comment]
@hazendaz is this tied to a JIRA issue?

**Comment by hazendaz** [comment]
Currently it is not.  I'll look at getting that taken care of.  Thanks.

**Comment by hazendaz** [comment]
ok - created jira ticket - RF-13964.  If this looks like it can go forward, I will go back through and give the maven updates another go.  That team is moving faster these days.  I can also squash commits into one and properly label the commit itself rather than just pull request.  Let me know.  Thanks.

**Comment by michpetrov** [comment]
Thanks, @hazendaz,
I tried the changes and it seems not have broken anything so it's good to go. What do you mean by "I will go back through and give the maven updates another go"? Are you going to make more updates? The commit messages should include the JIRA number but I can take care of that.

As for the last commit I don't think `.checkstyle` belongs in the repository, I'm going to delete the file after this is merged. You can remove the commit.

**Comment by hazendaz** [comment]
I can squash and get the JIRA ticket in place.  I noticed when rebasing the last time a number of maven plugins now out of date or missed the first time I went through this.  I've got those updated locally and ready to go but haven't uploaded yet.  Maybe it makes more sense for you to take this and I can do a separate pull request since you already took the time to verify this.  Let me now what you want me to do.  Thanks.

**Comment by michpetrov** [comment]
If you have more updates then push them, there's no reason to update the POMs only to update them again right after.

**Comment by michpetrov** [comment]
@hazendaz, is this is final version of the PR? I don't get notified on file changes so I'm not sure if you did any updates.

**Comment by hazendaz** [comment]
Yes this is final

--- Original Message ---

File: examples/irc-client/.checkstyle [hunk 1]
```diff
@@ -5,8 +5,12 @@
...
     <property name="checkstyle.header.file" value="/home/mpetrov/workspace/rf4-5/richfaces/examples/irc-client/target/checkstyle-header.txt"/>
   </local-check-config>
-  <fileset name="java-sources" enabled="true" check-config-name="maven-checkstyle-plugin" local="true">
-    <file-match-pattern match-pattern="src/main/java/.*\/.*\.java" include-pattern="true"/>
-    <file-match-pattern match-pattern="src/test/java.*\/.*\.java" include-pattern="true"/>
+  <local-check-config name="maven-checkstyle-plugin richfaces-checkstyle-report" location="jar:file:/C:/Users/Jeremy/.m2/repository/org/richfaces/richfaces-build-resources/4.5.1-SNAPSHOT/richfaces-build-resources-4.5.1-SNAPSHOT.jar!/richfaces-checkstyle/richfaces-checkstyle.xml" type="remote" description="maven-checkstyle-plugin configuration richfaces-checkstyle-report">
+    <property name="checkstyle.cache.file" value="${project_loc}/target/checkstyle-cachefile"/>
+    <property name="checkstyle.header.file" value="C:\Users\Jeremy\Documents\GitHub\richfaces\examples\irc-client\target\checkstyle-header-richfaces-checkstyle-report.txt"/>
```
Comment (ppitonak) [inline examples/irc-client/.checkstyle]: There is an absolute Windows path

**Pull request merged**